bundles/tools.vitruv.applications.umljava/src/tools/vitruv/applications/umljava/java2uml/JavaToUmlMethod.reactions

```diff
@@ -355,27 +355,27 @@ routine changeUmlReturnType(java::Method jMeth, java::TypeReference jType) {
 //=========================================== General
 //=========================================== 
 
-reaction JavaElementVisibilityChanged {
-    after element java::Modifier created and inserted in java::AnnotableAndModifiable[annotationsAndModifiers]
+reaction JavaElementVisibilityChangedAfterInsertion {
+    after element java::Modifier inserted in java::AnnotableAndModifiable[annotationsAndModifiers]
     with newValue instanceof Public || newValue instanceof Private || newValue instanceof Protected
-    call changeUmlNamedElementVisibility(affectedEObject, newValue)
+    call changeUmlNamedElementVisibility(affectedEObject)
 }
 
-reaction JavaElementMadePackagePrivate {
-    after element java::Modifier deleted and removed from java::AnnotableAndModifiable[annotationsAndModifiers]
+reaction JavaElementMadeVisibilityChangedAfterRemoval {
+    after element java::Modifier removed from java::AnnotableAndModifiable[annotationsAndModifiers]
     with oldValue instanceof Private
     || oldValue instanceof Public
     || oldValue instanceof Protected
-    call changeUmlNamedElementVisibility(affectedEObject, null)
+    call changeUmlNamedElementVisibility(affectedEObject)
 }
```

Comment:
The names of these Reactions might be improved. First, I guess that the "made" in `JavaElementMadeVisibilityChangedAfterRemoval` should not be there. Second, it is not that obvious to what "after insertion" and "after removal" refers. Maybe we could simply call them `JavaVisibilityModifierInserted` and `JavaVisibilityModifierRemoved`, as this is what the trigger specification matches?

Reply:
Thanks. Seems I was a bit too fast, adjusted the reaction names as you suggested.